umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/instrument.clj

```diff
@@ -82,42 +87,58 @@
 
 (defn xml-elem->instrument
   "Returns instrument record from the instrument element."
-  [doc base-xpath instrument-elem]
-  (let [child-instruments (get-child-instruments doc base-xpath instrument-elem)]
-    (util/remove-nil-keys
-     (if (> (count child-instruments) 0)
-       {:ShortName (value-of instrument-elem iso/short-name-xpath)
-        :LongName (value-of instrument-elem iso/long-name-xpath)
-        :Characteristics (char-and-opsmode/parse-characteristics instrument-elem)
-        :Technique (without-default-value-of instrument-elem "gmi:type/gco:CharacterString")
-        :NumberOfInstruments (count child-instruments)
-        :ComposedOf child-instruments
-        :OperationalModes (char-and-opsmode/parse-operationalmodes instrument-elem)}
-       {:ShortName (value-of instrument-elem iso/short-name-xpath)
-        :LongName (value-of instrument-elem iso/long-name-xpath)
-        :Characteristics (char-and-opsmode/parse-characteristics instrument-elem)
-        :Technique (without-default-value-of instrument-elem "gmi:type/gco:CharacterString")
-        :OperationalModes (char-and-opsmode/parse-operationalmodes instrument-elem)}))))
+  [doc base-xpath alt-xpath? instrument-elem]
+  (if alt-xpath?
+    ;; This is the NOAA case, where both the shortname and longname are combined
```

Comment:
Might be worth adding "See CMR-4885 for more details" to the end of this comment block

Reply:
Done.